### ISSUE TITLE
samzong ask join website reviewer

### ISSUE THREAD

**Issue by SAMZONG** [karmada-io/website#389]
Dear Karmada team,

I would like to express my gratitude for creating such an excellent project. As an open-source enthusiast and a cloud-native product manager, I have been following and participating in the development of the Karmada project from being a user to becoming a contributor.

I believe that I possess the necessary skills and experience to contribute to the Karmada community, including a deep understanding of multi-cluster management capabilities, Karmada's product vision, and the ability to make meaningful contributions to the open-source community.

My goal is to contribute to the Karmada community by maintaining product documentation, making it easier for others to contribute to Karmada. specifically:

* Tasks such as maintaining the build process for the document website
* synchronizing & review Chinese documentation
* writing English documentation for new features.
* Additionally, I am eager to contribute to the product design of Karmada.

In summary, my vision is to enable more people to use and contribute to Karmada through documentation, and to help more people understand the value and future potential of Karmada.

Thank you for considering my application. I look forward to contributing to the success of the Karmada project.

Best regards,

@SAMZONG

**Comment by SAMZONG** [comment]
## What I do

- Maintain `karmada-docs` channel in cncf slack. join us https://app.slack.com/huddle/T08PSQ7BQ/C05D72G0DFW
- PRs
  - #386
  - #385
  - #384
  - #383
  - #382
  - #381
  - #380
  - #366
  - #282
  - #204
  - #188
  - #179
  - #89

**Comment by RainbowMango** [comment]
Thanks @SAMZONG for all your contributions to the community. 
You have my full support, please send a PR to put your name on https://github.com/karmada-io/website/blob/main/OWNERS.

**Comment by SAMZONG** [comment]
https://github.com/karmada-io/community/pull/37
